react/week2/Task/src/Todo.js

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from 'react';
+import './Todo.css';
+import Task from './Task'
+
+
+
+ const Todo = () => {
+    
+    const [tasks, setTasks] = useState([
+        {
+            description: "Get out of bed",
+            completed: false
+        },
+        {
+            description: " brush teeth",
+            completed: false
+        },
+        {
+            description: "eat breakfast",
+            completed: false
+        }
+    ]);
+
+   
+
+
+    const addTask = () => {
+        const getRandomTask = tasks[Math.floor(Math.random() * tasks.length)];
+        const newTasks = [...tasks, { description:getRandomTask, completed: false }];
```

Comment:
Here you're saying `description: getRandomTask` (while `getRandomTask` is an object) and that's why when you try to add a task the app crashes with the `Objects are not valid as a React child` error. ❕